sr_watchdog/src/sr_watchdog/watchdog.py

```diff
@@ -36,20 +36,23 @@ class CheckResultWrongFormat(SrWatchdogExceptions):
 
 
 class SrWatchdog(object):
-    def __init__(self, tested_system_name="tested system", checks_classes_list=[]):
+    def __init__(self, tested_system_name="tested system", checks_classes_list=[], initial_wait_for_checks=20):
```

Comment:
should we set more time as default? e.g. 60, with bimanual real system might take longer to start

Reply:
Done in 0fa4de0